content/docs/1.6.0/best-practices.md

```diff
@@ -99,6 +99,46 @@ Any extra disks must be written in the `/etc/fstab` file to allow automatic moun
 
 Don't use a symbolic link for the extra disks. Use `mount --bind` instead of `ln -s` and make sure it's in the `fstab` file. For details, see [the section about multiple disk support.](../volumes-and-nodes/multidisk/#use-an-alternative-path-for-a-disk-on-the-node)
 
+### Disk Performance Optimization  
+
+Before configuring workloads, ensure that you have set up the following basic requirements for optimal disk performance.  
+
+- SATA/NVMe SSDs or disk drives with similar performance  
+- 10 Gbps network bandwidth between nodes  
+- Dedicated Priority Classes for system-managed and user-deployed Longhorn components  
+
```

Comment:
In 1.6, Longhorn will deploy a default priority class `longhorn-critical`, so we can change the wording as below.

"A dedicated Priority Class for system-managed and user-deployed Longhorn components. Longhorn will install a default Priority Class `longhorn-critical` by default."